<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPX Gcode to X3g Converter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gpxui.qrc">
    <normaloff>:/icons/gpx.ico</normaloff>:/icons/gpx.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gbFiles">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Input and Output Files</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelInput">
         <property name="text">
          <string>Input gcode:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editInput"/>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="tbtnInputGcode">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnTranslate">
         <property name="text">
          <string>Translate!</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbSettings">
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="4" column="0" colspan="2">
        <widget class="QWidget" name="widgetSettingsButtons" native="true">
         <layout class="QHBoxLayout" name="layoutSettingsButtons">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btnAdvancedToggle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Advanced &gt;&gt;</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnDefaults">
         <property name="text">
          <string>Restore Defaults</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnReload">
         <property name="text">
          <string>Reload</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QWidget" name="widgetVanillaSettings" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item row="0" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="labelMachineType">
            <property name="text">
             <string>Machine type:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboMachineType">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>10</number>
            </property>
            <property name="maxVisibleItems">
             <number>15</number>
            </property>
            <item>
             <property name="text">
              <string>Cupcake Gen3 XYZ, Mk5/6 + Gen4 Extruder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cupcake Gen4 XYZ, Mk5/6 + Gen4 Extruder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cupcake Pololu XYZ, Mk5/6 + Gen4 Extruder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cupcake Pololu XYZ, Mk5/6 + Pololu Extruder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Core-XY with HBP - single extruder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Core-XY with HBP - single extruder, slow Z</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Clone R1 Single with HBP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Clone R1 Dual with HBP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Replicator 1 - single extruder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Replicator 1 - dual extruder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Replicator 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Replicator 2 with HBP</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Replicator 2X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ZYYX - single extruder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ZYYX - dual extruder</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="labelGcodeFlavor">
            <property name="text">
             <string>Gcode flavor:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboGcodeFlavor">
            <item>
             <property name="text">
              <string>RepRap</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ReplicatorG/Makerbot Desktop</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="toolButton_5">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QWidget" name="widgetAdvanced" native="true">
         <layout class="QFormLayout" name="formLayout_2">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="1">
           <widget class="QCheckBox" name="cboxProgress">
            <property name="text">
             <string>Recompute build progress</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="cboxDitto">
            <property name="text">
             <string>Ditto printing</string>
            </property>
            <property name="tristate">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="cboxRecalc">
            <property name="text">
             <string>Recalculate extrusion</string>
            </property>
            <property name="tristate">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelNominal">
            <property name="text">
             <string>Nominal filament width:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QWidget" name="widgetNominal" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QDoubleSpinBox" name="dsbNominal">
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="hsNominal">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelActual">
            <property name="text">
             <string>Actual filament width:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QWidget" name="widgetActual" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QDoubleSpinBox" name="dsbActual">
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="hsActual">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelDensity">
            <property name="text">
             <string>Packing density:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QWidget" name="widgetDensity" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QDoubleSpinBox" name="dsbDensity">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="hsDensity">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGPX_Configuration">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuGPX_Configuration"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gpxui.qrc"/>
 </resources>
 <connections/>
</ui>
